### PR TITLE
fix some bugs in store functional

### PR DESCRIPTION
**Pull request by djaf77**

### CODE REVIEW

File: lib/hafriedlander/Peg/Parser/Basic.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ public function __construct($string) {
 
 	protected function isCallable($name) {
 		return $this->isCallableCache[$name]
-			?? ($this->isCallableCache[$name] = \is_callable($this, $name));
+			?? ($this->isCallableCache[$name] = \is_callable([$this, $name]));
```
Comment (smuuf) [inline lib/hafriedlander/Peg/Parser/Basic.php:25]: Wow, nice catch.

File: lib/hafriedlander/Peg/Parser/Basic.php [hunk 2]
```diff
@@ -156,13 +156,13 @@ public function store(&$result, $subres, $storetag = \null) {
 		$storecalled = \false;
 
 		foreach ($this->typestack($result['_matchrule']) as $type) {
-			if ($method = $this->isCallable($storetag ? "{$type}_{$storetag}" : "{$type}_{$subres['name']}")) {
+			if ($this->isCallable($method = $storetag ? "{$type}_{$storetag}" : "{$type}_{$subres['name']}")) {
```
Comment (smuuf) [inline lib/hafriedlander/Peg/Parser/Basic.php:159]: Is there any practical reason for this change, too?

Comment (djaf77) [reply lib/hafriedlander/Peg/Parser/Basic.php:159]: isCallable() returns boolean value, there are not callable ;)

**Comment by smuuf** [comment]
You're completely right, of course. Thank you!

**Pull request merged**